src/test/java/racingcar/domain/CarCollectionTest.java

```diff
@@ -0,0 +1,57 @@
+package racingcar.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.strategy.FixtureMoveStrategy;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarCollectionTest {
+    private CarCollection carCollection;
+    private final List<String> DEFAULT_CAR_NAME = List.of("pobi", "crong", "honux");
+
+    public void setUp(List<String> carName) {
```

Comment:
```suggestion
    @BeforeEach
    void setUp() {
```

매번 같은 이름을 사용하는데 junit의 setup 기능을 활용하는건 어떤가요?